lhotse/ais/batch_loader.py

```diff
@@ -0,0 +1,141 @@
+from typing import TYPE_CHECKING, Any, Dict, Iterator, Optional, Tuple
+
+if TYPE_CHECKING:
+    from aistore.sdk.batch.batch import Batch
```

Comment:
Please remove the global scope import. I prefer limited type annotations for 3rd party libs in cases like these.